pyproject.toml

```diff
@@ -22,12 +22,15 @@ classifiers = [
 python = ">=3.8, <3.11"
 grpcio = "^1.48.1"
 grpcio-tools = "^1.48.1"
+google-cloud = "^0.34.0"
+google-api-core = "^2.11.0"
+protobuf = "~4.21"
```

Comment:
@ashwinidulams can this work with `protobuf=3.20.x`? https://github.com/numaproj/numalogic/issues/123

Reply:
will test with 3.20.x